router/router.nginx.conf.in

```diff
@@ -76,6 +76,18 @@ server {
     listen [::]:80;
 }
 
+server {
+    server_name batch2-driver.*;
+    client_max_body_size 10000m;
```

Comment:
I thought you wanted to remove the client_max_body_size.

Reply:
Oops, missed that one.  Fixed, thanks!

Reply:
This one's still here. Did you intend to keep it?